script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -156,7 +156,7 @@ Yes, I have no opinion on who you choose to have a future relationship with, but
 [sha:8fe88f32d8e629f01c1284695efd2d5b499be606]{
 -- Page 66, Offset 19840.
 -- 「遠野家の長男が、その、どこの者とも判らない女性と夜な夜な逢引きをしているだなんて、そんな恥知らずな事はつつしむべきではないでしょうか？」
-"For the eldest son of the Tohno family to, well, rendezvous with an unknown woman every night. Shouldn't you refrain from such indecent acts?"
+"For the eldest son of the Tohno family to--- well, rendezvous with an unknown woman every night. Shouldn't you refrain from such indecent acts?"
```

Comment:
Script lint failed, revert?

Reply:
Ok, so what's happening here is that these are not the CJK dashes, they are normal dashes. And the parser sees the first two `--` and thinks ok, everything after this is a machine comment, I will just ignore it.

Replace them with proper CJK dashes and it'll fix the problem.